docs/section5/Rule5-43.md

```diff
@@ -0,0 +1,44 @@
+
+# Envelope - Rule 5-43  
+
+**Rule ID:** 5-43  
+**Rule Description:** The  proposed roof surfaces shall be modeled using the same solar reflectance as in the user model.  
+**Rule Assertion:** P-RMR SurfaceOpticalProperties:absorptance_solar_exterior = U-RMR SurfaceOpticalProperties:absorptance_solar_exterior  
+**Appendix G Section:** Section G3.1-1(a) Building Envelope Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. get_opaque_surface_type()
+  2. get_surface_conditioning_category()
+  3. match_data_element()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for P_RMR: `scc_dictionary_p = get_surface_conditioning_category(P_RMR)`
+
```

Comment:
I don't think we need to check space conditioning category.

Reply:
Removed.

Reply:
I don't think we need to get surface conditioning category - i.e. line 23 should be removed.

Reply:
That's right. get_surface_conditioning_category is now removed.